lib/js/components/Tile/Tile.stories.mdx

```diff
@@ -1,109 +0,0 @@
-import { Meta, Story, Props } from '@storybook/addon-docs';
-import Tile from 'design-system/lib/js/components/Tile';
-import { TILE_COLORS, TILE_STATES } from './Tile.consts';
-import { ICONS } from 'design-system/lib/js/components/Icon/Icon.consts';
-
-<Meta title="components/Tile" component={Tile} />
-
-#Tile
-
-**Tile component**
-
-<Story
-	name="interactive"
-	parameters={{
-		design: {
-			type: 'figma',
-			url: 'https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=1923%3A34378',
-		},
-	}}
-	args={{
-		interactive: true,
-		iconLeft: null,
-		iconRight: null,
-		text: 'this is a text text',
-		eyebrowText: 'this is an eyebrowText text',
-		additionalText: '',
-		isEyebrowTextUppercase: false,
-		state: TILE_STATES.DEFAULT,
-	}}
-	argTypes={{
-		iconLeft: {
-			control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
-			defaultValue: null,
-		},
-		iconRight: {
-			control: { type: 'select', options: [null, ...Object.keys(ICONS)] },
-			defaultValue: null,
-		},
-		color: {
-			control: { type: 'select', options: [...Object.values(TILE_COLORS)] },
-			defaultValue: TILE_COLORS.NEUTRAL,
-		},
-		state: {
-			control: { type: 'select', options: [...Object.values(TILE_STATES)] },
-			defaultValue: TILE_STATES.DEFAULT,
-		},
-	}}
->
-	>
-	{(args) => {
-		return {
-			components: { Tile },
-			props: Object.keys(args),
-			template:
-				'<tile :text="text" :eyebrowText="eyebrowText" :interactive="interactive" :iconLeft="ICONS[iconLeft]" :additionalText="additionalText" :iconRight="ICONS[iconRight]" :color="color" :is-eyebrow-text-uppercase="isEyebrowTextUppercase" :state="state" />',
-			created() {
-				this.ICONS = ICONS;
-			},
-		};
-	}}
-</Story>
-
-<Story
-	name="static"
```

Comment:
this story is missing after the migration

Reply:
Right, I added it.